blog_article/blog_article.adoc

```diff
@@ -0,0 +1,161 @@
+messaging brokers. It is widely used and supported by several broker implementations: RabbitMQ, ActiveMQ and SwiftMQ,
+to name just a few examples. The architecture that is assumed by AMQP 0.9.1 relies mainly on queues and exchanges. +
+In contrast to that, AMQP 1.0 was developed to be more universally applicable and therefore does not know concepts
+like these entities. By assuming less, the protocol should be more flexible to use. +
+As also stated on the AMQP website, the protocol was introduced "to become the standard protocol for
+interoperability between all messaging middleware". Using the RabbitMQ broker as an example, we want to take
+a closer look at the extent to which AMQP is supported and how different versions interoperate with each other.
+Therefore, in the following, we will compare the use of three Java client libraries: RabbitMQ, SwiftMQ and Qpid JMS.
+The RabbitMQ broker requires an activated AMQP 1.0 plugin for this. +
+The competing consumer and publish-subscribe patterns were implemented using these clients within this
+https://github.com/NovatecConsulting/showcase-rabbitmq/tree/main[repository]. Feel free to have a look at the implementations
+and how they differ from each other!
+
+== RabbitMQ Support for AMQP 1.0
+Most of the established brokers only or mainly support AMQP 1.0 since this protocol version was introduced as an OASIS
+standard. In contrast to that, RabbitMQ focuses on AMQP 0.9.1, and AMQP 1.0 support can only be added via a plugin.
+Nevertheless, since SwiftMQ, ActiveMQ and Qpid are JMS-based, there are fewer AMQP 1.0 client libraries available for
+programming languages other than Java. +
+In case a broker is needed which supports AMQP 0.9.1 and AMQP 1.0 on equal levels, the choice will remain limited:
+
+|===
+|Broker Implementation |AMQP Protocol Support
+
+|https://www.rabbitmq.com/[RabbitMQ]
+|mainly supports AMQP 0.9.1; AMQP 1.0 support by plugin
+
+|https://www.swiftmq.com/[SwiftMQ]
+|mainly supports AMQP 1.0; limited AMQP 0.9.1 support
+
+|https://activemq.apache.org/components/artemis/[ActiveMQ Artemis]
+|supports AMQP 1.0
+
+|https://activemq.apache.org/components/classic/[ActiveMQ "Classic"]
+|supports AMQP 1.0 via a transport connector
+
+|https://qpid.apache.org/components/broker-j/index.html[Qpid Broker-J]
+|supports AMQP 0.8 to 1.0
+
+|===
+
+Coming back to RabbitMQ as an example, their Java client only supports sending and receiving AMQP 0.9.1 messages.
+So another library is needed to use the AMQP 1.0 plugin. At the moment of writing this article and as also stated
+in the documentation of the plugin, there are only a few clients for AMQP 1.0, especially for Java:
+
+* https://www.swiftmq.com/docs/docs/client/amqp/[SwiftMQ]:
+The plugin's developers tested the functionalities mainly using SwiftMQ.
+* https://qpid.apache.org/releases/qpid-proton-j-0.33.8/[Qpid Proton-J]:
+The documentation is not that detailed and does not provide examples.
```

Comment:
After you analysed it and also after reading the goals of the project again, I think Proton is not a library for "business applications", but rather one for developing brokers or client libraries (https://github.com/apache/qpid-proton-j).
For example https://github.com/apache/qpid-jms and https://github.com/vert-x3/vertx-proton are using proton internally to exchange messages via AMQP 1.0.
It looks like some time ago Proton had its own Messaging API which eventually was removed (https://stackoverflow.com/questions/41536299/qpid-proton-j-deprecates-messenger-but-what-to-use-instead). The last release on Maven Central is 0.16.0. 
There are also examples for Proton, like https://github.com/apache/qpid-proton-j/tree/0.33.8/examples or also the qpid-jms or vertex-proton implementations. But as I already mentioned its very low level and not comparable to an AMQP Client like SwiftMQ.

Besides Qpid Proton-J, I also found Qpid Proton-J2 (https://qpid.apache.org/releases/qpid-protonj2-1.0.0-M2/). At the moment there are two contributes on GitHub working on it (https://github.com/apache/qpid-protonj2).
At first glance, it looks like this version provides an easier to use and maybe more intuitive API (at least to someone like me which is not that familiar with AMQP 1.0).